spec/views/recipe_foods/new.html.erb_spec.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
- [ ] This is a redundant test file, kindly take it out or write a view test in it

Reply:
@VICTOR-CHUKWU Please what do you mean by take it out?
delete the helper or what!

Reply:
it is suppose to be a spec file for your view recipe food right? And now there is no test in the file, so you don't need it  the best thing to do is to write your view test in it, I hope it is clear now